deploy/charts/hstream-operator/values.yaml

```diff
@@ -0,0 +1,43 @@
+# Default values for hstream-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: hstreamdb/hstream-operator-controller
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
```

Comment:
Looks this value is doesn't work ?

Reply:
these configuration items are referred in `_helpers.tpl`

Reply:
I mean, if user set `serviceAccount.create = false`, looks the serviceAccount still will created ?

Reply:
Yep, the serviceAccount will still been created even `serviceAccount.create` is set to `false`

Reply:
The ServiceAccount wouldn't been created if `serviceAccount.create` is set to false